widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -420,25 +441,57 @@ export function ConfirmWalletsModal(props: PropTypes) {
                             <div className="button__content">
                               <WalletIcon size={18} color="info" />
                               <Divider size={4} direction="horizontal" />
-                              <Typography variant="label" size="medium">
+                              <Typography
+                                variant="label"
+                                size="medium"
+                                color={
+                                  showCustomDestination
+                                    ? '$neutral600'
+                                    : undefined
+                                }>
                                 {i18n.t('Send to a different address')}
                               </Typography>
                             </div>
                             <ExpandedIcon
                               orientation={
                                 showCustomDestination ? 'up' : 'down'
                               }>
-                              <ChevronDownIcon size={10} color="gray" />
+                              <ChevronDownIcon size={10} color="secondary" />
                             </ExpandedIcon>
                           </CustomDestinationButton>
                         }
                         onClickTrigger={() =>
                           setShowCustomDestination((prev) => !prev)
                         }>
-                        <Divider size={4} />
                         <StyledTextField
-                          placeholder={i18n.t('Your destination address')}
+                          ref={customDestinationInputRef}
+                          style={{ padding: 0 }}
+                          autoFocus
+                          placeholder={i18n.t(
+                            'Enter {blockchainName} address',
+                            {
+                              blockchainName: getBlockchainDisplayNameFor(
+                                requiredWallet,
+                                blockchains
+                              ),
+                            }
+                          )}
                           value={customDestination || ''}
+                          suffix={
+                            customDestination ? (
+                              <IconButton
+                                onClick={handleClearCustomDestination}
+                                variant="ghost">
+                                <ClearIcon size={16} />
```

Comment:
<ClearIcon size={12} />
```suggestion
                                <ClearIcon size={16} />
```